docs/development/quickstart.rst

```diff
@@ -191,9 +191,13 @@ Creating and submitting a pull request
 
 You can contribute bug fixes, new features, and documentation updates by submitting a
 GitHub pull request (PR). This section will guide you through the process. We encourage
-you to `ask for help <https://www.astropy.org/help.html>`_ if you get stuck. The astropy
+you to `ask for help <https://www.astropy.org/help.html>`_ if you get stuck. The Astropy
 community is welcoming and friendly and will help you!
 
+If you are new to the Astropy Project and interested to submit a large patch
+(e.g., a new big feature or significant refactoring), we encourage you to first
+discuss your ideas on |astropy-dev mailing list| to increase the chance of your PR
+being accepted. We thank you for your patience!
```

Comment:
Thanks, good idea. I would remove the "We thank you for your patience!" - I think we're actually quite good in timely responses (but less good perhaps in having to deal with a very large PR, but this is just what you are trying to avoid here).

Reply:
Thanks for the review! That sentence has been removed.